src/Moryx.Resources.Interaction/Moryx.Resources.Interaction.csproj

```diff
@@ -5,6 +5,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Description>Resource with a REST endpoint for UIs to interact with the resource graph and instances.</Description>
     <CreatePackage>true</CreatePackage>
+    <PackageTags>MORYX;IIoT;IoT;UI;REST</PackageTags>
```

Comment:
This is no REST Library - it is a library with a resource to provide an endpoint for the resource management. 

Maybe: MORYX;IIoT;IoT;Resource